tests/unit/s2n_connection_size_test.c

```diff
@@ -45,7 +45,7 @@ int main(int argc, char **argv)
     }
 
     /* Carefully consider any increases to this number. */
-    const uint16_t max_connection_size = 4350;
+    const uint16_t max_connection_size = 4478;
```

Comment:
We currently allocate two buffers for received signature schemes, so any increase results in 2x the number of bytes allocated for each connection.
https://github.com/aws/s2n-tls/blob/eb168f2025372e6f299486a718f24999a3715705/tls/s2n_handshake.h#L108-L115

Reply:
I do hate to see this number go up, even temporarily. Could we skip any IANAs that we know we won't support, so that we can cap the array at the number of IANAs we support? Or does that complicate things too much for a quick fix?